### PR TITLE
Bumping datadog agent RAM to 512req / 1024 limit (from 256/512).

### PR DESCRIPTION
**Pull request by lasryaric**
## Description

Doing this because we see datadog agents being restarted for OOM and we are missing some datadog traces. Hoping this will fix it.

### CODE REVIEW

File: k8s/datadog-values.yml [hunk 1]
```diff
@@ -43,10 +43,10 @@ agents:
       resources:
         requests:
           cpu: 200m
-          memory: 256Mi
+          memory: 512Mi
         limits:
           cpu: 200m
-          memory: 512Mi
+          memory: 1024Mi
```
Comment (fontanierh) [inline k8s/datadog-values.yml:49]: On our setup it's actually useless to have a different limit from the request. With autopilot, it's the same thing

**Review by fontanierh — APPROVED** [review]
I think it's really weird that the daemon needs so much ram. OK to try it, but I think the issue might be something else :/

File: k8s/datadog-values.yml (hunk 1 above)
Comment (fontanierh) [reply k8s/datadog-values.yml:49]: Can you start by increasing request to 512 and leave the limit to 512 as well ?

**Comment by lasryaric** [comment]
>I think it's really weird that the daemon needs so much ram. OK to try it, but I think the issue might be something else :/

Looking at our traces and the number of spans they have, I am actually surprised that this can run with that little RAM.
I am debugging a Slack webhook issue, we generate ~1200 spans just for one webhooks, each with some payload 🙃

**Comment by fontanierh** [comment]
> So I have set both requests and limit to 512, hoping this is actually a bump and not just the same as before?

I hope too. If not we'll have to bump both to 1024 :/

**Comment by fontanierh** [comment]
@lasryaric 

> Looking at our traces and the number of spans they have, I am actually surprised that this can run with that little RAM.
I am debugging a Slack webhook issue, we generate ~1200 spans just for one webhooks, each with some payload 🙃

this thing is really just forwarding stuff to the cluster agent... so shouldn't need that much

**Comment by lasryaric** [comment]
Yes but it's receiving so much data, it must go somewhere. Running any sorts of app today takes a few hundred megabytes of ram already.

**Pull request merged**